src/common.rs

```diff
@@ -14,13 +18,16 @@ macro_rules! define_float_funcs {
         => $lname:ident/$lfname:ident;
     )+) => {
         #[cfg(not(feature = "std"))]
-        pub(crate) trait FloatFuncs : Sized {
+        pub trait FloatFuncs : Sized + common_sealed::Sealed {
```

Comment:
This should have some docs now that it's public

Reply:
Should be fixed in [85ad5eb](https://github.com/linebender/kurbo/pull/332/commits/85ad5eb179ebcf2c6c28f775ab482a8143f223f8)